test/instructions/from.test.ts

```diff
@@ -448,319 +448,319 @@ describe("FROM", () => {
         assert.equal(from.getPlatformFlag(), null);
     });
 
-    it("FROM privateregistry.com/base/image", () => {
```

Comment:
you might not want to take this change, dropping any individual commit is trivial.

It was a bit of an adventure writing this commit. `npm build` does not clean up the out tests directory -- I had deleted most of the tests so that I could focus on a single test, but it was still testing them.

Reply:
> you might not want to take this change, dropping any individual commit is trivial.

@jsoref Where do I find the steps to do this?

Reply:
oh, you can just ask me, or since it's the last commit, just make a branch w/o it and merge that. Or you could use `git cherry-pick ...` the commits you like. (There are too many ways to do things w/ git...)

For me, it's:

```
git rebase -i master
```
followed by deleting individual commits, save+quit my editor + `git push jsoref --force-with-lease`

Reply:
@jsoref My mistake. I thought there was a command that I could use (by leaving a comment in GitHub).

Reply:
Well, you can visit https://github.com/rcjsuen/dockerfile-ast/commits/9de6bb4cf8dbd1b1ff9445d2669809441051fe97~, click the `Switch branches/tags` widget, enter a new name, and select `Create branch: ... from ...` and then create a PR from it.

Based on your response though, lemme drop that commit.